enrolments/tests/views/test_session.py

```diff
@@ -48,6 +54,25 @@ def test_get_session(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(payload, SessionDetailSerializer(self.session).data)
 
+    def test_create_session(self):
+        url = reverse("session-list")
+        self.client.force_authenticate(self.user)
+        request = {
+            "name": "Fall 2020",
+            "start_date": date(2020, 12, 1),
+            "fields": [1, 2],
+            "classes": [
+                {
+                    "name": "Test Class Create",
+                    "days": [Class.MONDAY, Class.WEDNESDAY],
+                    "location": "Waterloo",
+                    "facilitator": self.user.id,
+                }
+            ],
+        }
+        response = self.client.post(url, request, format="json")
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
```

Comment:
This is failing with 400 != 201
It works as expected in postman though ://

Reply:
i think similar idea to the above comment - if we pass in ids of fields that exist this should be good